interop/tapir/src/main/scala/caliban/interop/tapir/TapirAdapter.scala

```diff
@@ -114,6 +120,8 @@ object TapirAdapter {
         case _                                                              => false
       }
     )
+    val isSSE          =
+      request.header(HeaderNames.Accept).map(v => v.contains(MediaType.TextEventStream.toString())).getOrElse(false)
```

Comment:
Consider extracting `MediaType.TextEventStream.toString()` into a `val` to avoid conversion on each request

Reply:
Also maybe use `.exists` instead of `.map(...).getOrElse(false)`